### DISCUSSION TITLE
YASKAWA INFORM3 translation problems

### DISCUSSION THREAD

**Discussion by robertocognoli** [HALRobotics/Beta#132] [Q&A]
Hello everyone, 
does anyone use yaskawa on hal robotic in grasshopper?
I have some problems with the translation of the code.
I can do the simulation but not the code export.
does anyone have some sample file?

**Comment by thibaultschwartz** [comment]
Hi Roberto,
What do you mean by "having problems with code export" ? 
Did you follow the tutorial? https://docs.hal-robotics.com/Grasshopper/6-Control/Contents.html#62-export-a-procedure

**Comment by robertocognoli** [comment]
Thank you. yes i've worked with HAL with another brand of robot.
i've some issue due to the specific model of robot.  the problem is the controller setting.
So I'm looking for some YASKAWA user that works with HAL to understand better the workflow.
